include/gatekeeper_net.h

```diff
@@ -323,6 +329,98 @@ ipv6_if_configured(struct gatekeeper_if *iface)
 	return !!(iface->configured_proto & GK_CONFIGURED_IPV6);
 }
 
+/* Return the L2 header length of a packet to be sent out on @iface. */
+static inline size_t
+pkt_out_l2_hdr_len(struct gatekeeper_if *iface)
```

Comment:
Why not make it as an attribute of ``struct gatekeeper_if``, so that we don't need to call this function to re-calculate it?